client/my-sites/sidebar-unified/style.scss

```diff
@@ -483,7 +483,7 @@ $font-size: rem( 14px );
 			}
 		}
 
-		.sidebar__menu.is-togglable:not( .is-toggle-open ):hover {
+		.sidebar__menu.is-togglable:not( .is-toggle-open ).hovered {
```

Comment:
I would consider a comment here to explain why we're not using CSS and what handles application of `.hovered` class.

Reply:
Done d5cca73